src/swing.ts

```diff
@@ -80,9 +80,10 @@ export interface Stack {
    * Creates an instance of Card and associates it with an element.
    *
    * @param {HTMLElement} element
+   * @param {boolean} prepend
    * @return {Card}
    */
-  createCard(elment: HTMLElement): void;
+  createCard(elment: HTMLElement, prepend: boolean): void;
```

Comment:
@mashhoodr @rajeevriitm Sure thing. That's what I've got on my local branch anyway so I can push it up this evening if you can wait that long?